src/Utilities/Math.hpp

```diff
@@ -162,3 +164,22 @@ template <typename T>
 constexpr T sgn(const T& val) {
   return sgn_detail::sgn(val, std::is_signed<T>{});
 }
+
+/// \ingroup UtilitiesGroup
+/// \brief Raises a double to the integer power n.
+BLAZE_ALWAYS_INLINE double integer_pow(const double x, const int e) {
```

Comment:
`BLAZE_ALWAYS_INLINE` isn't defined here in some setups.  `SPECTRE_ALWAYS_INLINE` would be the one for this file, although a plain `inline` is probably more appropriate in this case.